app/main.py

```diff
@@ -1,3 +1,46 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: int) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int) -> Distance:
+        return Distance(km=self.km * other)
+
+    def __truediv__(self, other: int) -> Distance:
+        return Distance(km=round(self.km / other, 2))
+
+    def __lt__(self, other: int) -> bool:
+        return self.km < other
+
+    def __gt__(self, other: int) -> bool:
+        return self.km > other
+
+    def __eq__(self, other: int) -> bool:
+        return self.km == other
+
+    def __le__(self, other: int) -> bool:
+        return self.km <= other
+
+    def __ge__(self, other: int) -> bool:
+        return self.km >= other
```

Comment:
![image](https://github.com/mate-academy/py-distance-class/assets/103201010/effbb03e-f285-45f9-9281-266c2d4801e8)
What if `other` is of type `Distance` here and in all the following methods?